tests/conformance/conformance_test.go

```diff
@@ -2444,6 +2444,83 @@ var internalTestCases = []testCase{
 		compatScratchConfig: types.OptionalBoolTrue,
 	},
 
+	{
+		name: "add--exclude-includes-star",
+		dockerfileContents: strings.Join([]string{
+			"# syntax=docker/dockerfile:1.9-labs",
+			"FROM scratch",
+			"ADD --exclude=**/*-c ./ subdir/",
+		}, "\n"),
+		contextDir:          "dockerignore/populated",
+		fsSkip:              []string{"(dir):subdir:mtime"},
+		compatScratchConfig: types.OptionalBoolFalse,
+		dockerUseBuildKit:   true,
+	},
+
+	{
+		name: "add--exclude-includes-slash",
+		dockerfileContents: strings.Join([]string{
+			"# syntax=docker/dockerfile:1.9-labs",
+			"FROM scratch",
+			"ADD --exclude=*.txt / subdir/",
+		}, "\n"),
+		contextDir:          "dockerignore/populated",
+		fsSkip:              []string{"(dir):subdir:mtime"},
+		compatScratchConfig: types.OptionalBoolFalse,
+		dockerUseBuildKit:   true,
+	},
+
+	{
+		name: "add--exclude-includes-dot",
+		dockerfileContents: strings.Join([]string{
+			"# syntax=docker/dockerfile:1.9-labs",
+			"FROM scratch",
+			"ADD --exclude=*-c . subdir/",
+		}, "\n"),
+		contextDir:          "dockerignore/populated",
+		fsSkip:              []string{"(dir):subdir:mtime"},
+		compatScratchConfig: types.OptionalBoolFalse,
+		dockerUseBuildKit:   true,
+	},
+	{
+		name: "copy--exclude-includes-subdir-slash",
+		dockerfileContents: strings.Join([]string{
+			"# syntax=docker/dockerfile:1.9-labs",
+			"FROM scratch",
+			"COPY --exclude=**/*-c / subdir/",
+		}, "\n"),
+		contextDir:          "dockerignore/populated",
+		fsSkip:              []string{"(dir):subdir:mtime"},
+		compatScratchConfig: types.OptionalBoolFalse,
+		dockerUseBuildKit:   true,
+	},
+
+	{
+		name: "copy--exclude-includes-dot-slash",
+		dockerfileContents: strings.Join([]string{
+			"# syntax=docker/dockerfile:1.9-labs",
+			"FROM scratch",
+			"COPY --exclude='!**/*-c' ./ subdir/",
```

Comment:
(Turns out I was mistaken, and quoting does work for the value of the `--exclude=` argument.)